README.md

```diff
@@ -189,6 +189,21 @@ Some scenarios (`CreateDeleteClients` and `CrawlUsers`) require a service accoun
    * assign the roles `manage-clients` and `view-users`
 * the client secret to be passed to the tests can be copied from the `Credentials` tab
 
+#### Scenario `keycloak.scenario.admin.CreateRealms`
+
+This scenario requires the following Client settings, in addition to the requirements above:
+
+* In to the `Scopes` tab
+    * Set `Full Scope Allowed` to `Off`
+    * Add role `create-realm` in the `Realm Roles` section
+* In the `Service Account Roles` tab
+    * assign the roles `create-realm` in the `Realm Roles` section
+
+The `Full Scope Allowed` flag is especially important to disable, because if enabled,
+the access token will contain *all* all the accesses to all the realms,
```

Comment:
Remove a second "all" from the description.

```suggestion
the access token will contain *all* the accesses to all the realms,
```

Reply:
Fixed. Thanks for catching this one!